djsf.json

```diff
@@ -291,6 +291,17 @@
 							"description": "The number of the next train in case the train returns as another train number. This can be used to directly select the next train when the train arrives at its destination station.",
 							"id": "#nextTrainNumber",
 							"required":false
+						},
+						"multiTractionTrains" : {
+							"type":"array",
+							"description": "This could be null, an empty array or an array of objects. Null means that this trains does not have any multitraction partner. An empty array means that this trains is a multitraction partner for another leading train. And an array of objects means that this train has multitraction partner and it's the leading train.",

```

Comment:
Please add: The order of the trains in the array defines the order of trains in the multitraction.